src/relayedTransactionV2Builder.spec.ts

```diff
@@ -0,0 +1,41 @@
+import {loadTestWallets, TestWallet} from "./testutils";
+import {RelayedTransactionV2Builder} from "./relayedTransactionV2Builder";
+import {Address} from "./address";
+import {TransactionPayload} from "./transactionPayload";
+import {assert} from "chai";
+
+describe("test relayed v2 transaction", function () {
+    let alice: TestWallet, bob: TestWallet;
+
+    before(async function () {
+        ({alice, bob} = await loadTestWallets());
+    });
+
+    it("should compute relayed v2 tx", async function () {
+        this.timeout(20000);
```

Comment:
[question]

Is this 20 seconds? If so, why such long time? Is it an integration test?